src/sync.js

```diff
@@ -1361,6 +1361,15 @@ SyncApi.prototype._processSyncResponse = async function(
         const currentCount = data.device_one_time_keys_count.signed_curve25519 || 0;
         this.opts.crypto.updateOneTimeKeyCount(currentCount);
     }
+    if (this.opts.crypto && data.device_unused_fallback_keys) {
```

Comment:
Hmm, so a supporting server will list your fallback key in every sync response indefinitely? It would nice if there were some way for it to only appear when changed, rather than every time, but I'll leave that to the spec and server discussion.

Reply:
Added a [comment](https://github.com/matrix-org/matrix-doc/pull/2732#discussion_r489294662) to the MSC for this.